packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardSettingsTab.tsx

```diff
@@ -126,12 +236,7 @@ export const YppDashboardSettingsTab = () => {
             label="Category of imported videos"
             description="Choose a category to be assigned to the imported videos by default. You can change it for each video later once it’s imported."
           >
-            <Select
-              disabled={currentChannel?.shouldBeIngested}
-              items={categoriesSelectItems}
-              onChange={setCategory}
-              value={category}
-            />
+            <Select items={categoriesSelectItems} onChange={setCategoryId} value={categoryId} />
```

Comment:
Shouldn't we disable the Select when the category is already chosen?

https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=3542%3A89393&t=YX2PO34nrAi7tXy2-0

Reply:
We changed that, and you should be able to update the category.